lib/presentation/widget/shared_space_document/shared_space_document_widget.dart

```diff
@@ -386,15 +386,16 @@ class _SharedSpaceDocumentWidgetState extends State<SharedSpaceDocumentWidget> {
               context,
               workGroupNodes,
               _moreActionList(context, documents),
-              _removeWorkGroupNodeAction(workGroupNodes, itemSelectionType: ItemSelectionType.multiple)))
+              SharedSpaceOperationRole.deleteNodeSharedSpaceRoles.contains(_arguments.sharedSpaceNode.sharedSpaceRole.name) ? _removeWorkGroupNodeAction(workGroupNodes, itemSelectionType: ItemSelectionType.multiple) : SizedBox.shrink()))
           .build();
   }
 
   List<Widget> _moreActionList(BuildContext context, List<WorkGroupNode> workGroupNodes) {
     return [
       if (Platform.isIOS) _exportFileAction(workGroupNodes, itemSelectionType: ItemSelectionType.multiple),
       if (Platform.isAndroid) _downloadFilesAction(workGroupNodes, itemSelectionType: ItemSelectionType.multiple),
-      _copyToAction(context, workGroupNodes, itemSelectionType: ItemSelectionType.multiple)
+      _copyToAction(context, workGroupNodes, itemSelectionType: ItemSelectionType.multiple),
+      if (SharedSpaceOperationRole.duplicateNodeSharedSpaceRoles.contains(_arguments.sharedSpaceNode.sharedSpaceRole.name)) _duplicateMultipleSelection(workGroupNodes),
```

Comment:
why not 
```
if (SharedSpaceOperationRole.duplicateNodeSharedSpaceRoles.contains(_arguments.sharedSpaceNode.sharedSpaceRole.name)) {
  _duplicateMultipleSelection(workGroupNodes),
}
```

Reply:
same for other

Reply:
I don't think it's possible in this case, dart recognizes it as a collection when I add brackets

Reply:
Don't think it acts as a normal if else statement